README.md

```diff
@@ -32,16 +26,12 @@ stage        | pantheon/internal-staging/fedora/#
 
 In the table above, # is the fedora build number (19, 20, 22). Note that drush is only installed on app servers, and there are no app servers on anything prior to f22; therefore, at the moment, we are only building for f22.
 
-To release a new version of drush, simply update the VERSIONS.txt file and commit. Run `make` to build locally with docker, or `make with-native-tools` to build without docker. Push to one of the branches above to have an official RPM built and pushed to Package Cloud via Circle CI.
+To release new versions of drush, simply update the VERSIONS.txt file and commit. Run `make` to build locally with docker, or `make with-native-tools` to build without docker. Push to one of the branches above to have an official RPM built and pushed to Package Cloud via Circle CI.
 
 ## Provisioning drush on Pantheon
 
 Pantheon will automatically install any new RPM that is deployed to Package Cloud. This is controlled by [pantheon-cookbooks/drush](https://github.com/pantheon-cookbooks/drush/blob/master/recipes/default.rb).
 
 ## drush RPM versioning strategy
 
-If there is ever a drush version 1.x, then we will maintain both drush-0.x and drush-1.x RPMs, so that legacy WordPress sites can continue to run 0.x until ready to upgrade.
-
-If drush ever releases a 0.x build that accidentally breaks backwards compatibility with old Pantheon sites, then we simply will not install that version until the bug is fixed in a future 0.x release.  If drush ever makes a permanent change to the 0.x line that breaks backwards compatibility with old Pantheon sites without bumping its version up to 1.x, then we will simply need to make another RPM named something other than '-0.x' or '-1.x'.
-
-
+If drush ever releases a x.x build that accidentally breaks backwards compatibility with old Pantheon sites, then we simply will not install that version until the bug is fixed in a future x.x release.  If drush ever makes a permanent change to the x.x line that breaks backwards compatibility with old Pantheon sites without bumping up its version number, then we will simply need to make a new RPM.
```

Comment:
This section can be deleted. It is only here for wp-cli because wp-cli won't make a stable release. This describes how we will handle various situations that may come up.  Drush is making stable releases, so we don't need to document our strategy -- we'll just follow normal SemVer conventions.